test/runFIT.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash -e
+
+############################################
+# Global variable
+############################################
+RACKHD_TEST_DIR="$( cd -P "$( dirname "$0" )" && pwd )"
+
+#############################################
+#
+# Usage
+############################################
+Usage(){
+    set +x
+    echo "Function: This script is used to set up environment for FIT and run FIT."
+    echo "Usage: $0 [OPTIONS]"
+    echo "  OPTIONS:"
+    echo "    Mandatory options:"
+    echo "      -p, --PASSWORD: password of current user which is used for FIT to log into host by, it's required."
+    echo "    Optional options:"
+    echo "      -g, --TEST_GROUP: test group of FIT, such as imageservice, smoke"
+    echo "      -s, --TEST_STACK: target test stack of FIT, such as docker, vagrant..."
+    echo "      -v, --TEST_LOG_LEVEL: log level of FIT, 0 is the most verbose for log output , 9 for most succinct"
```

Comment:
Actually 0 is the least verbose.  you will get no output other than pass/fail with log level 0.
9 is full on logging including info/debug etc.  A good mix of console/file output is at level 4  or 5. Note that logs files are also being auto created into the log_output directory for every run.  We should look to copy those off at some point.